### PR TITLE
RCIAM-414_SSPmod_attrauthcomanage_fix_configuration_parser

### PR DESCRIPTION
**Pull request by ioigoume**
Refactor validation of configuration parameters.

### CODE REVIEW

File: lib/Auth/Process/COmanageDbClient.php [hunk 1]
```diff
@@ -1282,6 +1136,135 @@ private function getTermsAgreement($personId, $coTermsId)
...
+                ),
+                'voRoles' => array(
+                    'type' => 'is_array'
+                ),
+                'urnAuthority' => array(
+                    'type' => 'is_string'
+                ),
+                'registryUrls' => array(
+                    'type' => 'is_array',
+                    'key_list' => array(
```
Comment (NicolasLiampotis) [inline lib/Auth/Process/COmanageDbClient.php:1239]: `key_list` should be handled as the required list of keys

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]

File: lib/Auth/Process/COmanageDbClient.php (hunk 1 above)
Comment (ioigoume) [reply lib/Auth/Process/COmanageDbClient.php:1246]: done

**Review by ioigoume — COMMENTED** [review]
Completed changes

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**